### PR TITLE
Persisting celery

### PR DESCRIPTION
**Pull request by nanuxbe**

### CODE REVIEW

File: server/webapp/dbmodels.py [hunk 1]
```diff
@@ -360,13 +362,16 @@ class WorkLogDb(db.Model):  # pylint: disable=R0903
...
     project_id = db.Column(UUID(True), db.ForeignKey('projects.id'))
+    parset_id = db.Column(UUID(True))
+    result_id = db.Column(UUID(True), default=None)
     start_time = db.Column(db.DateTime(timezone=True), server_default=text('now()'))
     stop_time = db.Column(db.DateTime(timezone=True), default=None)
     status = db.Column(work_status, default='started')
     error = db.Column(db.Text, default=None)
 
-    def __init__(self, project_id, work_type=None):
+    def __init__(self, project_id, parset_id = parset_id, work_type=None):
```
Comment (Ashalynd) [inline server/webapp/dbmodels.py]: The rest should be filled in automatically.

Comment (nanuxbe) [inline server/webapp/dbmodels.py]: yes, but parset_id=parset_id?  there is no global var named parset_id

Comment (Ashalynd) [inline server/webapp/dbmodels.py]: There is such property in that table.

Comment (Ashalynd) [inline server/webapp/dbmodels.py]: Ah right, I see what you mean. Indeed, you are right!

**Pull request merged**